library/ns1_zone.py

```diff
@@ -176,9 +176,35 @@
     description:
       - To enable slaving of your zone by third party DNS servers,
         you must include a primary object.
-    type: str
+    type: dict
     required: false
     default: None
+    suboptions:
+      enabled:
+        description:
+          - If true the zone the zone is enabled for outgoing zone transfers.
+        type: bool
+        required: false
+      secondaries:
+        description:
+          - Collection of secondary DNS servers that slave off this zone.
+        type: list
+        required: false
+        suboptions:
+          ip:
+            description:
+              - The IPv4 address of the secondary DNS server.
+            type: str
+          port:
```

Comment:
should there be a default on this?

Reply:
Since it's optional, I don't think it needs a default val.  The API handles setting the default if this field is omitted in the request.